content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -7,16 +7,44 @@ page_title: "Upgrades - Infrastructure Administration - Terraform Enterprise"
 
 This section explains how to upgrade Terraform Enterprise to a new
 version. [Learn more about availability during upgrades here](../system-overview/reliability-availability.html#availability-during-upgrades).
+## Before Upgrade
```

Comment:
@laurapacilio could you help to review this section?
I have just added it as a pre-caution that users should be aware of the responsibility for the backup task.
I wonder if it should be more explicit that we strongly recommend creating a backup copy of the storage prior to upgrade.